libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/JsCallbackReceiver.java

```diff
@@ -159,6 +161,10 @@ public void executeCallback(String callbackId, String params) {
 
                 mListener.onLinkTapped(url, title);
                 break;
+            case CALLBACK_VIDEOPRESS_INFO_REQUEST:
+                // Extract the VideoPress id from the callback string
+                mListener.onVideoPressInfoRequested(params.substring(3));
```

Comment:
Could use a comment explaining what `substring(3)` is doing. It would also be good to protect against an `IndexOutOfBoundsException` here if params isn't as long as expected.